### PR TITLE
Add tutorial for multiclass classification

### PR DESCRIPTION
**Pull request by fgrosa**

### CODE REVIEW

File: tutorials/hipe4ml_tutorial_multiclass.ipynb [hunk 1]
```diff
@@ -0,0 +1,541 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "JZadboiAhjRm"
+   },
+   "source": [
+    "A _total set_ is built by merging the indivisual sets. A label is assigned to each candidate of the total set: 0 for pions, 1 for kaons, 2 for protons, 3 for electrons, and 4 for deuterons. The total set is split into a _train set_ and a _test set_, used for training and testing the algorithm, respectively. The fraction of the total set used as test set is defined by the _test_size_ parameter of the function [_sklearn.model_selection.train_test_split_](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html)"
```
Comment (fmazzasc) [inline tutorials/hipe4ml_tutorial_multiclass.ipynb:174]: indivisual -> individual

File: tutorials/hipe4ml_tutorial_multiclass.ipynb [hunk 2]
```diff
@@ -0,0 +1,541 @@
...
+    "!pip install uproot\n",
+    "!pip install xxhash\n",
+    "!pip install lz4"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### **Restart Runtime(temporary)** \n",
```
Comment (fmazzasc) [inline tutorials/hipe4ml_tutorial_multiclass.ipynb:62]: Is this still needed?

**Review by fmazzasc — APPROVED** [review]
Very nice!

File: tutorials/hipe4ml_tutorial_multiclass.ipynb (hunk 2 above)
Comment (fcatalan92) [reply tutorials/hipe4ml_tutorial_multiclass.ipynb:62]: I tested it and it is still needed.

File: tutorials/hipe4ml_tutorial_multiclass.ipynb (hunk 1 above)
Comment (fgrosa) [reply tutorials/hipe4ml_tutorial_multiclass.ipynb]: done!

File: tutorials/hipe4ml_tutorial_multiclass.ipynb (hunk 2 above)
Comment (fgrosa) [reply tutorials/hipe4ml_tutorial_multiclass.ipynb:62]: Thanks for testing it, indeed it is still needed

**Pull request merged**